src/adapters/icon_manager.js

```diff
@@ -84,4 +84,13 @@ export function createIcon(iconOptions, name, hoverEffect = false) {
   return element.firstElementChild;
 }
 
+export function createDefaultMapIcon() {
```

Comment:
great ! Can we set image dimension as parameters ?

Reply:
and `src` ? As I guess we will need to use other icons :framed_picture:

Reply:
Yup, good idea. It was a quick way of getting an image blob from this particular SVG but it sure looks like something that should be generic :)